src/fsharp/CompileOps.fs

```diff
@@ -1749,13 +1749,21 @@ type IRawFSharpAssemblyData =
     abstract HasAnyFSharpSignatureDataAttribute : ILGlobals -> bool
     abstract HasMatchingFSharpSignatureDataAttribute : ILGlobals -> bool
 
-type IProjectReference = 
+type TimeStampCache() = 
+    let cacheCreation = DateTime.Now
```

Comment:
Sorry for being late, but using `DateTime.UtcNow` everywhere not only a faster way, but mitigates UTC adjustments problems https://blogs.msdn.microsoft.com/kirillosenkov/2012/01/10/datetime-utcnow-is-generally-preferable-to-datetime-now/